src/components/MessagePreview.tsx

```diff
@@ -0,0 +1,81 @@
+import TimePassed from "./TimePassed";
+
+interface MessagePreviewProps {
+  previewProps: {
+    clientName: string;
+    clientImage: string;
+    unreadCount: number;
+    lastMessageAt: number;
+    lastMessage: string;
+  };
+}
+
+const MessagePreview: React.FC<MessagePreviewProps> = ({ previewProps }) => {
+  return (
+    <div
+      className="
+        relative
+        w-[304px]
+        h-[52px]
+        rounded-[12px]
+        flex
+        gap-[8px]
+        pt-[8px] pr-[12px] pb-[8px] pl-[8px]
+        bg-alt-grey
+        text-border-grey
+        text-sm
+      "
+    >
+      <img
+        src={previewProps.clientImage}
```

Comment:
nit: It'd be good to be able to fallback to our 'placeholder' icon here in case the server doesn't send us a proper image.